tracing-subscriber/src/fmt/time/chrono_crate.rs

```diff
@@ -39,26 +73,53 @@ mod tests {
     use crate::fmt::format::Writer;
     use crate::fmt::time::FormatTime;
 
+    #[cfg(feature = "chrono")]
+    use super::ChronoFmtType;
     #[cfg(feature = "chrono")]
     use super::ChronoLocal;
     #[cfg(feature = "chrono")]
     use super::ChronoUtc;
 
     #[cfg(feature = "chrono")]
     #[test]
-    fn test_chrono_format_time_utc() {
+    fn test_chrono_format_time_utc_default() {
         let mut buf = String::new();
         let mut dst: Writer<'_> = Writer::new(&mut buf);
-        assert!(FormatTime::format_time(&ChronoUtc, &mut dst).is_ok());
+        assert!(FormatTime::format_time(&ChronoUtc::default(), &mut dst).is_ok());
         // e.g. `buf` contains "2023-08-18T19:05:08.662499+00:00"
     }
 
     #[cfg(feature = "chrono")]
     #[test]
-    fn test_chrono_format_time_local() {
+    fn test_chrono_format_time_utc_custom() {
+        let fmt = ChronoUtc {
+            format: ChronoFmtType::Custom("%a %b %e %T %Y".to_owned()),
+        };
         let mut buf = String::new();
         let mut dst: Writer<'_> = Writer::new(&mut buf);
-        assert!(FormatTime::format_time(&ChronoLocal, &mut dst).is_ok());
+        assert!(FormatTime::format_time(&fmt, &mut dst).is_ok());
+        println!("{}", buf);
```

Comment:
probably a leftover, but i'd remove this.

Reply:
fixed!